crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -144,8 +144,12 @@ def end_step(success = true, errors = {})
           errors: errors
         }
         if current_step == upgrade_steps_6_7.last && success
-          # Mark the end of the upgrade process
+          # Mark the end of the upgrade process and cleanup the progress
           FileUtils.rm_f running_file
+          progress[:current_substep] = :end_of_upgrade
+          progress[:current_substep_status] = :finished
+          progress[:current_nodes] = {}
+          progress[:current_node_action] = "finished"
```

Comment:
I added additional condition in the UI to not display this last action. I don't mind leaving it as is.